assets/javascripts/kitten/components/molecules/buttons/button-group/index.js

```diff
@@ -20,34 +17,34 @@ const StyledButtonGroup = styled.div`
     width: auto;
 
     &:not(:last-child) {
-      margin-right: -${borderSize};
+      margin-right: ${pxToRem(-2)};
```

Comment:
les border sont de 1px maintenant, je pense qu'on peut passer la valeur à 1, tu en penses quoi ?

```suggestion
      margin-right: ${pxToRem(-1)};
```

Reply:
Yes, bien vu ! Je l'ai fait sur certains